From cf43e36206dfd153557c881c110bb9249f739162 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 11 Dec 2022 10:39:22 +0100
Subject: [PATCH 13/16] power: supply: rk818-battery: Don't reset dsoc to 0 on
 rk818_vb_low_irq

The interrupt fires at 3.4V vbat which is not equivalent to 0% charge.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/power/supply/rk818_battery.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/power/supply/rk818_battery.c b/drivers/power/supply/rk818_battery.c
index 1ed2d0c24669..c3dab11a42e0 100644
--- a/drivers/power/supply/rk818_battery.c
+++ b/drivers/power/supply/rk818_battery.c
@@ -2930,7 +2930,6 @@ static irqreturn_t rk818_vb_low_irq(int irq, void *bat)
 {
 	struct rk818_battery *di = (struct rk818_battery *)bat;
 
-	di->dsoc = 0;
 	rk_send_wakeup_key();
 	BAT_INFO("lower power yet, power off system! v=%d, c=%d, dsoc=%d\n",
 		 di->voltage_avg, di->current_avg, di->dsoc);
-- 
2.43.0

